package.json

```diff
@@ -1,20 +1,20 @@
 {
   "name": "uncrustify",
-  "version": "0.6.1",
+  "version": "0.64",
   "description": "A highly configurable, easily modifiable source code beautifier for C, C++, C#, ObjectiveC, D, Java, Pawn and VALA",
   "main": "uncrustify",
   "scripts": {
-      "preinstall": "./configure && make"
+      "preinstall": "mkdir build; cd build; cmake .."
```

Comment:
With this cmake will run even if mkdir and / or cd fail.
This should fix it: `mkdir -p build && cd build && cmake ..` 
The make step is missing too but I am not sure if this should be inside a script called preinstall in the first place.